tsung/tsung-basic.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0"?>
+<!DOCTYPE tsung SYSTEM "/usr/local/src/tsung/tsung-1.0.dtd" [
+    <!ENTITY validation SYSTEM "transactions/validation.xml">
+    <!ENTITY save_form SYSTEM "transactions/save_form.xml">
+    <!ENTITY authentication SYSTEM "transactions/authentication.xml">
+    <!ENTITY new_case SYSTEM "transactions/new_case.xml">
+    <!ENTITY update_cases SYSTEM "transactions/update_cases.xml">
+    <!ENTITY update_sync_record SYSTEM "transactions/update_sync_record.xml">
+]>
+<tsung loglevel="info">
+    <clients>
+        <client host="localhost" use_controller_vm="true"/>
+    </clients>
+
+    <servers>
+        <server host="10.10.1.28" port="5432" type="tcp"></server>
+    </servers>
+
+    <sessions>
+        <!-- Form submission broken down in primitives based on: https://docs.google.com/document/d/1MzQSjIyWhICRxnyEdGuW9eigD0N0Ka6p7DWcLP81204/edit#heading=h.ckpofgcu6eec -->
+        <session probability="30" name="form-submission-new-case" type="ts_pgsql">
+            <!-- Set the case -->
+            <setdynvars sourcetype="file" fileid="casedb-new" delimiter="," order="iter">
```

Comment:
I haven't had a chance to look at this In detail but generally looks good.
I was also thinking about the CSV approach but only for cases that already
exist so that we can simulate case updates.

Definitely sounds like a good idea to create data for be cases in erlang.
Should be fun, I've never liked at erlang before.